GitCommands/Git/GitCommandHelpers.cs

```diff
@@ -684,6 +684,30 @@ public static IReadOnlyList<GitItemStatus> GetStatusChangedFilesFromString(IGitM
             }
         }
 
+        private static string RemoveWarnings(string statusString)
```

Comment:
Is this tested via `TestGetStatusChangedFilesFromString`?

Reply:
yes, I updated the testcases, some existed before too